x/src/test.rs

```diff
@@ -17,18 +17,34 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+    #[structopt(name = "TESTNAME")]
+    testname: Option<String>,
+    #[structopt(last = true)]
+    args: Vec<String>,
 }
 
 pub fn run(args: Args, config: Config) -> Result<()> {
+    let pass_through_args = args
+        .testname
+        .as_ref()
+        .into_iter()
+        .map(|s| s.as_str())
+        .chain(iter::once("--"))
```

Comment:
We add `--` and `--` is also specified in the circle config. Does clap eat this?

Reply:
Yeah if you use the "last" option for an argument it makes it so that you have to specify `--` in order to supply that arg, but clap eats that`--` so we need to add it back in before we pass it to cargo.